File: cinema-backend/src/main/java/org/example/cinemabackend/cinema/core/domain/SeatZone.java
package org.example.cinemabackend.cinema.core.domain;

import java.math.BigDecimal;

public enum SeatZone {
    STANDARD(27.90),
    VIP(35.90),
    PROMO(19.90),
    WHEELCHAIR(19.90),
    CORRIDOR(null);

    private final Double price;

    SeatZone(Double price) {
        this.price = price;
    }

    public BigDecimal getPrice() {
        return BigDecimal.valueOf(price);
    }
}

File: cinema-customer-frontend/src/app/repertory/components/buy-ticket/enums/seat-zone.ts
export enum SeatZone {
  STANDARD = 'STANDARD',
  VIP = 'VIP',
  PROMO = 'PROMO',
  WHEELCHAIR = 'WHEELCHAIR',
  CORRIDOR = 'CORRIDOR'
}

export const SeatPrices: { [key in SeatZone]: string } = {
  [SeatZone.STANDARD]: '27,90 ZŁ',
  [SeatZone.VIP]: '35,90 ZŁ',
  [SeatZone.PROMO]: '19,90 ZŁ',
  [SeatZone.WHEELCHAIR]: '19,90 ZŁ',
  [SeatZone.CORRIDOR]: ''
};

export const SeatPricesAsNumbers: { [key in SeatZone]: number } = {
  [SeatZone.STANDARD]: 27.90,
  [SeatZone.VIP]: 35.90,
  [SeatZone.PROMO]: 19.90,
  [SeatZone.WHEELCHAIR]: 19.90,
  [SeatZone.CORRIDOR]: 0
};
